### PR TITLE
send through the hash of commitment complete on reveal

### PR DESCRIPTION
**Pull request by maxtaco**
- this will help in replaying flips after the fact

### CODE REVIEW

**Comment by maxtaco** [comment]
@mmaxim not sure where I should be commiting this (to master instead?) but this adds eventual support for replaying games after the fact

File: go/chat/flip/prot.avdl [hunk 1]
```diff
@@ -69,11 +69,23 @@ protocol flip {
...
+  record UserDeviceCommitment {
+    UserDevice ud;
+    Commitment c;
+  }
+
+  fixed Hash(32);
+
+  record Reveal {
+    Secret secret;
+    Hash cch; // Hash of comimtment complete (sent in prior round).
```
Comment (mmaxim) [inline go/chat/flip/prot.avdl]: Typo

**Comment by maxtaco** [comment]
@mmaxim is it actually OK if I merge this into master instead of merging into your branch?  because i need to do other stuff on top of it.  eh, whatever works for you

**Comment by mmaxim** [comment]
Yeah just send everything to master

**Comment by maxtaco** [comment]
oh no, i can't, since it's build on top of your types integration work.

**Pull request merged**